omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
```

Comment:
@guzzilar The reason of this condition has been provided in the description of the commit, 0924c74f2c824d8cfa2e0e65d488daf1683a0072.